spinalcordtoolbox/scripts/sct_detect_pmj.py

```diff
@@ -239,6 +236,18 @@ def extract_sagital_slice(self):
 
         run_proc(['sct_crop_image', '-i', self.fname_im, '-zmin', str(self.rl_coord), '-zmax', str(self.rl_coord + 1), '-o', self.slice2D_im])
 
+    def extract_pmjsymmetrical_sagittal_slice(self):
```

Comment:
to follow naming logic i would suggest `extract_pmj_symmetrical_sagittal_slice `

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3468/commits/9e9c7b3a074faa3bdb0474bcea2eeb30220474d4